R/module_init_data.R

```diff
@@ -53,7 +53,7 @@ srv_init_data <- function(id, data) {
 
   moduleServer(id, function(input, output, session) {
     logger::log_debug("srv_data initializing.")
-    # data_rv contains teal_data object
+    # data contains teal_data object
```

Comment:
`data` in this comment rather refers to the `data_out` but I think the whole comment can be removed